mono/metadata/object.c

```diff
@@ -62,8 +62,9 @@ mono_ldstr_metadata_sig (MonoDomain *domain, const char* sig, MonoError *error);
 static void
 free_main_args (void);
 
+// In order to share code, error is optional.
```

Comment:
For code that don't care about the `MonoError`, use the following pattern (as also pointed out by https://github.com/mono/mono/pull/6268#pullrequestreview-84275755):

```
char *
mono_string_to_utf8_image_ignore (MonoImage *image, MonoString *s)
{
	MonoError error;
	error_init(&errror);
	char *result = mono_string_to_utf8_internal (NULL, image, s, &error);
	mono_error_cleanup(&error); // FIXME don't swallow the error
	return result;
}